README.md

```diff
@@ -95,7 +95,7 @@ valkey-server --loadmodule /path/to/libjson.so
 2. Execute Valkey command:
     MODULE LOAD /path/to/libjson.so
 ```
-## Supported  Module Commands
+## Supported Module Commands
```

Comment:
I don't usually make commits to drop spaces... this one seems like a pretty good candidate for dropping...